ext/php8/span.h

```diff
@@ -14,25 +14,23 @@ static const int ddtrace_num_error_tags = 3;
 struct ddtrace_dispatch_t;
 
 struct ddtrace_span_t {
-    zval *span_data;
+    zend_object std;
+    zval properties_table_placeholder[5];
     uint64_t trace_id;
     uint64_t parent_id;
     uint64_t span_id;
     uint64_t start;
-    union {
-        uint64_t duration_start;
-        uint64_t duration;
-    };
+    uint64_t duration_start;
+    uint64_t duration;
```

Comment:
What is the motivation for removing the `union`?

Reply:
Because the userland interface wants to distinguish between finished and unfinished spans; now we can just return duration == 0, then userland knows unfinished

Reply:
Ah, makes sense. Thanks for clarifying.